src/main/kotlin/no/nav/familie/ef/iverksett/simulering/SimuleringKlient.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.familie.ef.iverksett.simulering
+
+import no.nav.familie.ba.sak.common.BaseService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.nav.familie.kontrakter.felles.simulering.DetaljertSimuleringResultat
+import no.nav.familie.log.NavHttpHeaders
+import no.nav.familie.log.mdc.MDCConstants
+import no.nav.security.token.support.client.core.oauth2.OAuth2AccessTokenService
+import no.nav.security.token.support.client.spring.ClientConfigurationProperties
+import org.slf4j.MDC
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import org.springframework.web.client.exchange
+import java.net.URI
+
+private const val OAUTH2_CLIENT_CONFIG_KEY = "familie-oppdrag-clientcredentials"
+
+@Service
+class SimuleringKlient(
+        @Value("\${FAMILIE_OPPDRAG_API_URL}")
```

Comment:
Savner `FAMILIE_OPPDRAG_API_URL` i config, og trenger azure-oppsett for familie-oppdrag
https://github.com/navikt/familie-ef-sak/blob/master/src/main/resources/application.yml#L35-L43
`FAMILIE_OPPDRAG_SCOPE` finnes i secreten `FAMILIE` som hentes inn ved oppstart, bare si ifra hvis du ønsker att jeg forklarer oppsettet :)

Reply:
Gjorde et første forsøk